test/integration.test.js

```diff
@@ -3,71 +3,69 @@
 'use strict'
 
 import { join } from 'path'
-import next from '../dist/server/next'
 import pkg from '../package.json'
+import next from '../dist/server/next'
+import {expectElement, setup, render, teardown} from 'next-test-helper'
```

Comment:
I hope we need to expose some low level helpers rather these generic helpers.
For an example, here are few cases:

* Get the programmatic API
* Run an app by giving the path
* Get a random available port

There are many ways, we could write tests. It's depend on test case to case.
Specially with Next.js integration tests, it gets so crazy. Sometimes, we may need to do `npm install`.